db/setup_notifications.sql

```diff
@@ -45,9 +45,5 @@ CREATE INDEX `notification_recipients_ix1` ON `notification_recipients` (`recipi
 
 -- changes for added notifications on oc shares
 
-ALTER TABLE oc_share ADD notify_uploads BOOL DEFAULT false;
+ALTER TABLE oc_share ADD notify_uploads INTEGER DEFAULT 0 NOT NULL;
```

Comment:
Keep in mind this would require changes in the web and reva too!

Reply:
That's why I said in #4061 that we need to test. The code in Reva does not seem to depend on the type, that is for now we can use the integer as a bool. Not sure about web.

Reply:
I'm not 100% sure, I remember doing `strconv.ParseBool` in some places but not sure about all of them. I guess as long as we only store 0 or 1 and we can parse it, it should be OK.

Reply:
Anyway given the change of table name, the whole thing is to be retested after merging cs3org#4061 (which you should review ;) ).